packages/frontend-api/src/Model/Complaint/ComplaintApiFacade.php

```diff
@@ -108,10 +108,10 @@ public function createFromComplaintInputArgument(Argument $argument): Complaint
         $orderCustomerUser = $order->getCustomerUser();
 
         $isSameCustomerUser = $customerUser !== null && $orderCustomerUser === $customerUser;
-        $isCustomerAccountOwner = $this->security->isGranted(CustomerUserRole::ROLE_API_ALL) &&
+        $hasAccessToAllCompanyOrders = $this->security->isGranted(CustomerUserRole::ROLE_API_COMPANY_ORDERS_VIEW) &&
```

Comment:
I'm thinking few things:
- should I cannot be able to create the complaint without permission to see the orders?
- isn't this limiting user with create order permission (and create complaint) from creating complaint?
- should it be responsibility of facade?

Reply:
implemented in c3142bc1c7